content/en/docs/reference/config-api/apiserver-encryption.v1.md

```diff
@@ -94,7 +94,13 @@ Each key has to be 32 bytes long for AES-CBC and 16, 24 or 32 bytes for AES-GCM.
 <thead><tr><th width="30%">Field</th><th>Description</th></tr></thead>
 <tbody>
     
-  
+<tr><td><code>apiversion</code><br/>
+<code>string</code>
+</td>
+<td>
+   <p>apiversion is the API Version for KMS provider (Allowed values: v2, empty or anything else will default fallback to v1)</p>
```

Comment:
```suggestion
   <p>apiVersion is the API Version for KMS provider (Allowed values: v2, v1 or empty. Any other value will result in an error.)</p>
```

nit: As part of the implementation we decided to default empty to `v1` and in validation we check if it's either `v1` or `v2`. Any other API version will result in an error.

ref: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/apis/config/validation/validation.go#L225-L232